lib/config.js

```diff
@@ -1,15 +1,33 @@
 'use strict';
 
+const gitRepoInfo = require('git-repo-info');
+
 module.exports = class AddonDocsConfig {
-  shouldDeploy() {
-    return true;
+  constructor() {
+    this.repoInfo = gitRepoInfo();
+  }
+
+  shouldUpdateLatest() {
+    if ('ADDON_DOCS_UPDATE_LATEST' in process.env) {
+      return process.env.ADDON_DOCS_UPDATE_LATEST !== 'false';
+    }
+
+    return !!this.repoInfo.tag;
```

Comment:
Thing that occurred to me as I was writing the documentation: we probably want to change this default behavior not to update `/latest` for prerelease tags. I think `node-semver` has an easy method for checking this on a parsed version string.